hazelcast/src/main/java/com/hazelcast/internal/config/ConfigValidator.java

```diff
@@ -240,6 +240,28 @@ private static void warnForDeprecatedHotRestartProp(MapConfig mapConfig,
         }
     }
 
+    public static void warnForUsageOfDeprecatedSymmetricEncryption(Config config, ILogger logger) {
+        String warn = "Symmetric encryption is deprecated and will be removed in a future version. Consider using TLS instead.";
+
+        if (config.getNetworkConfig() != null
```

Comment:
Could you also add a check which of networkings is enabled to prevent doubled warning display?
```
usesAdvancedNetworkConfig = config.getAdvancedNetworkConfig().isEnabled();
```

Reply:
Done